molgenis-security-ui/src/main/resources/l10n/security-ui_en.properties

```diff
@@ -2,7 +2,7 @@ security-ui-group-attribute-name-name=Group Identifier
 security-ui-member-details-header=Member details
 security-ui-btn-add-member=Add member
 security-ui-member-removing=Removing
-security-ui-members-page-title=Group members
+security-ui-members-page-title=Group
```

Comment:
What happens on upgrade of existing instances? I guess the text won't be updated?

Reply:
I suggest to create a story on how to migrate these values